### ISSUE TITLE
Refactor Maestro workshop

### ISSUE THREAD

**Issue by zachlatta** [hackedu/hack-camp#260]
Refactor our existing Maestro workshop for cohort 4.

It's totally fine if you decide to rewrite it or change what's created in the workshop. As long as we're introducing JavaScript and tying it into web development.

**Comment by jonleung** [comment]
Looks good!

**Comment by jonleung** [comment]
I need to create a PR for this.

Then I need to create a new task to have people review the workshop to rid bugs as discussed on wed aug 5th.

**Comment by maxwofford** [comment]
@jonleung Is this a high leverage enough task while we are working on the Labs program?

**Comment by zachlatta** [comment]
Has the work been done and all we're waiting on is having a PR submitted to
get the work merged?

On Wed, Aug 19, 2015 at 11:01 AM, Max Wofford notifications@github.com
wrote:

> Is this a high leverage enough task while we are working on the Labs
> program?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/hackedu/hack-camp/issues/260#issuecomment-132722620.

## 

Zach Latta
hackEDU | zach@hackedu.us

**Comment by jonleung** [comment]
@MaxWofford, I agree with you. I don't think this is a high leverage task while we are working on the Labs program. I am moving this issue to backlog.

**Details:**

I refactored the Cohort 3 Maestro workshop to Cohort 4's Maestro workshop (as can be seen [here](https://github.com/hackedu/hack-camp/blob/master/cohort_4/playbook/workshops/maestro/README.md)).

There are problems in Cohort 4's Maestro workshop. The problem now is that there is a large reading section to get through before you can write a single line of Maestro code. This makes the workshop boring. The solution is to make sure the student gets a chance to write Maestro code at a more frequent interval throughout the tutorial.

I have not begun to fix any of these problems yet. So in regards to @zachlatta's question:

> Has the work been done and all we're waiting on is having a PR submitted to
> get the work merged?

No. There hasn't.

**Comment by jonleung** [comment]
Because I don't think this is a high leverage task while we are working on the Labs program, I've actually decided to close this issue because:
1. one of our objectives for this week is to close all the Hack Camp issues
2. don't know when I'd get to this task
